.drone.yml

```diff
@@ -498,6 +498,70 @@ steps:
       - name: deps
         path: /go
 
+---
+kind: pipeline
+type: docker
+name: testing-e2e
+
+platform:
+  os: linux
+  arch: amd64
+
+depends_on:
+  - compliance
+
+trigger:
+  event:
+    - pull_request
+
+services:
+  - name: pgsql
+    pull: default
+    image: postgres:10
+    environment:
+      POSTGRES_DB: testgitea-e2e
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_INITDB_ARGS: --encoding=UTF8 --lc-collate='en_US.UTF-8' --lc-ctype='en_US.UTF-8'
+
+steps:
+  - name: deps-frontend
+    image: node:16
+    pull: always
+    commands:
+      - make deps-frontend
+
+  - name: build-frontend
+    image: node:16
+    commands:
+      - make frontend
+    depends_on: [deps-frontend]
+
+  - name: deps-backend
+    image: golang:1.18
+    pull: always
+    commands:
+      - make deps-backend
```

Comment:
Should add a `deps` volume so subsequent steps don't re-download go dependencies, see other pipelines for examples.

Reply:
Yeah, I'll add that back in. I removed it when I started running into the `EOF` error in CI:
```
FAIL    code.gitea.io/gitea/tests/e2e [setup failed]
# code.gitea.io/gitea/tests/e2e
modules/indexer/bleve/batch.go:8:2: github.com/blevesearch/bleve/v2@v2.3.2: unexpected EOF
FAIL
make[1]: *** [/drone/src/Makefile:666: e2e.pgsql.test] Error 1
make: *** [Makefile:8: test-e2e-pgsql] Error 2
```
I thought it fixed it at first, but now I don't think so. 
I think I might have to make this test run after the integration tests. Otherwise, both tests will be installing dependencies simultaneously on the same system.